src/fvOptions/actuatorLineSource/actuatorLineElement/actuatorLineElement.C

```diff
@@ -623,6 +630,24 @@ const Foam::scalar& Foam::fv::actuatorLineElement::momentCoefficient()
 }
 
 
+const Foam::scalar& Foam::fv::actuatorLineElement::tangentialCoefficient()
+{
+    return tangentialCoefficient_;
```

Comment:
I think we should rename these as `refTangentialCoefficient_` and `refNormalCoefficient_` for consistency and generality.